### PR TITLE
biofabricator

### PR DESCRIPTION
**Pull request by Ilya246**
relies on #23318

## About the PR
adds the biofabricator - a building that can make animal cubes from biomass
- is a T2 biochemical research which also unlocks the biomass reclaimer
- if emagged, can make hostile mobs, though they also cost plasma

nerfs cow biomass output
## Why / Balance
it could be interesting to have a way to inefficiently make mobs without having to order them

## Media

https://github.com/space-wizards/space-station-14/assets/57039557/28c4e70a-dfb5-4f36-a2aa-8772f2ee0821

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Science may now research the Biofabricator, which can make animal cubes from biomass!

### CODE REVIEW

**Comment by Ilya246** [comment]
i'd like maintainer approval on this, but i want the hostile mobs it makes to always be ghost roles *not aligned with their creator* so you can make a fish army
could make only fish have that too

File: Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml:66]: description copy pastes arent needed

**Comment by Ilya246** [comment]
PR essentially ready, only sprite remains
i can't sprite so i will either have to wait for someone to make one or steal something from tg

**Comment by K-Dynamic** [comment]
Interesting, but biomass reclaimers got removed from all maps a while back and have to be found on salvage missions

**Comment by Ilya246** [comment]
> Interesting, but biomass reclaimers got removed from all maps a while back and have to be found on salvage missions

the tech also lets you build them

**Comment by Ilya246** [comment]
ready for review

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:583]: biomass inherits the RawMaterial tag so there isn't a point in specifying a new tag for it. The lathe will automatically handle the material whitelist.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:599]: yes i know the indentation on these files is absolutely fucked but the lists are over indented by one

Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:599]: leave the cockroach in the regular recipes. I think it's funny

File: Resources/Prototypes/Recipes/Lathes/rehydrateable.yml
Comment (EmoGarbage404) [inline Resources/Prototypes/Recipes/Lathes/rehydrateable.yml:74]: all of these plasma amounts are kinda high IMO. could totally just be half for all of these.

File: Resources/Prototypes/tags.yml
Comment (EmoGarbage404) [inline Resources/Prototypes/tags.yml:51]: mentioned this above but the tag isn't needed

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
holy shit the first new biochemical technology in _6 fucking months._ Simply amazing.

Glad to see the biomass reclaimer back. It's such a meme.

File: Content.Server/Lathe/LatheSystem.cs
Comment (EmoGarbage404) [reply Content.Server/Lathe/LatheSystem.cs:159]: left proper review on the other pr.

File: Resources/Prototypes/Recipes/Lathes/rehydrateable.yml
Comment (Ilya246) [reply Resources/Prototypes/Recipes/Lathes/rehydrateable.yml:74]: ok, it's fishening time then

**Comment by Ilya246** [comment]
review addressed, ready for merge once i rebase onto master after the fix this relies on is merged

**Comment by Ilya246** [comment]
rebased, should be all good

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**
